src/pages/[platform]/deploy-and-host/fullstack-branching/pr-previews/index.mdx

```diff
@@ -44,4 +40,91 @@ With fullstack previews, you can set up ephemeral fullstack environments on ever
 1. The pull request preview is deployed on Amplify and available at `pr-1.appid.amplifyapp.com`.
 1. Once the pull request is merged into `main`, the request is closed and the fullstack environment is also automatically torn down.
 
-Once this feature is available, we will update this page with instructions for generating fullstack previews so you can view and validate proposed changes to your fullstack app before merging pull requests.
+### Prerequisites
+
+Before you get started, make sure you have the following:
+
+- [An Amplify Gen 2 app created](https://docs.amplify.aws/gen2/start/quickstart/)
```

Comment:
This link needs to be updated since we no longer use gen2 paths anymore.

Should it be this:
```suggestion
- [An Amplify Gen 2 app created](/[platform]/start/quickstart/)
```

Reply:
Done!